sdk/digitaltwins/digital-twins-core/tests.yml

```diff
@@ -1,13 +1,13 @@
 trigger: none
 
-extends:
-  template: ../../../eng/pipelines/templates/jobs/archetype-sdk-integration.yml
-  parameters:
-    PackageName: "@azure/digital-twins-core"
-    ResourceServiceDirectory: digitaltwins
-    TestBrowser: true
-    TestSamples: false
-    EnvVars:
-      AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
-      AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
-      AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
+stages:
+  - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+    parameters:
+      PackageName: "@azure/digital-twins-core"
+      ServiceDirectory: digitaltwins
+      MatrixFilters:
+        - TestType=node|browser
```

Comment:
Given you are overriding the default matrix filter don't you also need to disable the min/max testing here? Assuming you want to match the older behavior which might have not be correct.

Reply:
Ah, good catch.